src/numbers.js

```diff
@@ -1,45 +1,203 @@
 const add = (a, b) => {
```

Comment:
This module is wrong, almost in all these cases you wrote unnecessary logical statements. You should think about the generic case